pkg/webhook/patch.go

```diff
@@ -182,6 +183,43 @@ func addEnvironmentVariable(pod *corev1.Pod, envName, envValue string) patchOper
 	return patchOperation{Op: "add", Path: path, Value: value}
 }
 
+func addCommandArgs(pod *corev1.Pod, app *v1beta1.SparkApplication) []patchOperation {
+	i := 0
+
+	// Find the driver or executor container in the pod.
+	for ; i < len(pod.Spec.Containers); i++ {
+		if pod.Spec.Containers[i].Name == sparkDriverContainerName ||
+			pod.Spec.Containers[i].Name == sparkExecutorContainerName {
+			break
+		}
+	}
+
+	path := fmt.Sprintf("/spec/containers/%d", i)
+	var podSpec *v1beta1.SparkPodSpec
```

Comment:
Looks like the Travis build was not completed successfully, please do some dummy changes to trigger another build, e.g., add an empty line before 198 and remove the empty line after 198.

Reply:
@liyinan926 
Done.